src/CoherenceBuild/PackagePublisher.cs

```diff
@@ -13,64 +14,26 @@ namespace CoherenceBuild
 {
     public static class PackagePublisher
     {
-        public static void PublishToShare(
```

Comment:
We're copying packages early on to the artifacts dir so this is no longer necessary.